lib/rpush/daemon/dispatcher/apns_http2.rb

```diff
@@ -0,0 +1,49 @@
+module Rpush
+  module Daemon
+    module Dispatcher
+      class ApnsHttp2
+
+        URLS = {
+          production: 'https://api.push.apple.com:443',
+          development: 'https://api.development.push.apple.com:443'
+        }
+
+        DEFAULT_TIMEOUT = 60
+
+        def initialize(app, delivery_class, _options = {})
+          @app = app
+          @delivery_class = delivery_class
+
+          url = URLS[app.environment.to_sym]
+          @client = NetHttp2::Client.new(url,
+            ssl_context:     prepare_ssl_context,
+            connect_timeout: DEFAULT_TIMEOUT)
+        end
+
+        def dispatch(payload)
+          @delivery_class.new(@app, @client, payload.batch, payload.notification).perform
+        end
+
+        def cleanup
```

Comment:
Does http2 offer something similar to `@http.shutdown` called in `Push::Daemon::Dispatcher::Http`?